CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 Unreleased Changes
 ------------------
 
```

Comment:
Nit: Mention dropping of support for rails 7.0 here.

Reply:
It says
* Feature - Depend on `aws-sessionstore-dynamodb ~> 3` which depends on `rack ~> 3` and is not supported by Rails `7.0`.